docker/Dockerfile

```diff
@@ -1,62 +1,24 @@
 FROM continuumio/miniconda3
 
-MAINTAINER Gilles Bodart <gillesbodart@users.noreply.github.com>
+# Install build-essential (compiler and development tools)
+RUN apt-get update && \
+    apt-get install -y build-essential && \
+    rm -rf /var/lib/apt/lists/*
 
-RUN conda create -n env python=3.6
+RUN conda create -n env python=3.9
```

Comment:
Choosing the Python version is not something to be done lightly. The issue references Python 3.7 and the repository also indicates that LASER is compatible with Python >= 3.7. So why 3.9 here? Are there any specific reasons for this?

Reply:
Yes the issue referenced python 3.7, but the new `laser_encoders` library also indicates >=3.8 in the [readme](https://github.com/facebookresearch/LASER/blob/MLH-dev/laser_encoders/README.md). I used 3.9 because I thought it falls in range, no strong reason in particular. If Python 3.8 is preferred, I can switch.